pwnlib/tubes/ssh.py

```diff
@@ -910,14 +910,17 @@ def func(): pass
 
         func_src  = inspect.getsource(func).strip()
         setuid = True if setuid is None else bool(setuid)
+        
+        # Converts the environment variables to a list of tuples to remain order.
+        env = list(env.items())
 
         script = r"""
 #!/usr/bin/env python2
 import os, sys, ctypes, resource, platform, stat
 from collections import OrderedDict
```

Comment:
Looks like this was planned before since there's an import to `OrderedDict` but it's never used.